This pull complements a change for solving #1265

jotatsu opened the pull request:
This creates a window without borders or title in puttyng mode, as such, no corrections needs to be made when is integrated inside a form in DPS

jotatsu committed:
Update WINDOW.C

kmscode commented on WINDOWS/WINDOW.C:
```diff
@@ -725,6 +725,9 @@ int WINAPI WinMain(HINSTANCE inst, HINSTANCE prev, LPSTR cmdline, int show)
 	if (conf_get_int(conf, CONF_sunken_edge))
 	    exwinmode |= WS_EX_CLIENTEDGE;
+#ifdef PUTTYNG
+	winmode &= ~(WS_POPUPWINDOW);
```
I haven't run or tested this yet... But...

I think we should only modify this when puttyng is started from a connection. So:
if (hwnd_parent != 0)
  winmode...

This way if puttyng is started manually or from options/advanced, we get a full/proper window.

jotatsu replied:
I've added in c284855... the condition. Now if launched manually its normal with window title and left click menu. When hwnd_parent is present , it will be a borderless window without title

jotatsu committed:
Added condition for direct launch

Direct launch now is normal, embed window has no borders and fit into the mremote panel.
